api/graphite.go

```diff
@@ -1025,14 +1044,34 @@ func (s *Server) graphiteTagFindSeries(ctx *middleware.Context, request models.G
 		return
 	default:
 	}
-	seriesNames := make([]string, 0, len(series))
-	for _, serie := range series {
-		seriesNames = append(seriesNames, serie.Pattern)
+
+	switch request.Format {
+	case "lastupdate-json":
+		seriesVals := make([]models.SeriesLastUpdate, 0, len(series))
+		for _, serie := range series {
+			var lastUpdate int64
+			for _, node := range serie.Series {
+				for _, ndef := range node.Defs {
+					if ndef.LastUpdate > lastUpdate {
+						lastUpdate = ndef.LastUpdate
+					}
+				}
+			}
+			seriesVals = append(seriesVals, models.SeriesLastUpdate{Series: serie.Pattern, Ts: lastUpdate})
```

Comment:
i'm a bit confused here. doesn't this only include the series name (without tags) ? I thought you wanted it broken down by tags.
also, it looks like this doesn't include meta tags.

Reply:
This returns the same string that is currently returned, which is formatted like `name;tag1=val1;tag2=val2`

Reply:
@replay shouldn't we expand the name here to include meta tags?

Reply:
I can't think of a good reason why it would be necessary to include meta tags here, except for consistency so that whoever uses the endpoint also gets to see the meta tags. 
Note that in order to include meta tags in this result the `index/find_by_tags` endpoint and the `MemoryIdx.FindByTag()` method would all have to be modified to include meta tags in the returned results, which would slow finds down because those results would need to be enriched. So if we want to enable the user to also retrieve the meta tags on this endpoint we should make this an optional feature of the `index/find_by_tags` endpoint which by default is off.

Reply:
Yeah, I would say including meta tags here would be a separate feature.

Reply:
Actually, I just looked at the related code again, and the series returned by `clusterFindByTag` already have the meta tags if the feature is enabled. They're in the `.MetaTags` property. So if that's wanted, it would be easy to also add them.